tests/SetCookieTest.php

```diff
@@ -27,67 +28,135 @@ public function test_getters()
         $secure = true;
         $httpOnly = true;
         $sameSite = 'lax';
-        $setCookie = new SetCookie('name', 'value', $expiresAt, $path, $domain, $secure, $httpOnly, $sameSite);
-        $this->assertSame($expiresAt, $setCookie->expiresAt());
-        $this->assertSame($path, $setCookie->getPath());
-        $this->assertSame($domain, $setCookie->getDomain());
-        $this->assertSame($secure, $setCookie->isSecure());
-        $this->assertSame($httpOnly, $setCookie->isHttpOnly());
-        $this->assertSame($sameSite, $setCookie->getSameSite());
+        $name = 'me_likey';
+        $value = 'cookies';
+        $setCookie = new SetCookie($name, $value, $expiresAt, $path, $domain, $secure, $httpOnly, $sameSite);
+        self::assertSame($name, $setCookie->getName());
```

Comment:
All PHPUnit assertion methods are actually static, it's wrong to call then as if were an object.
https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Assert.php#L1579

Reply:
PS: Please note that the `expectedException` methods are non static though.

Reply:
Why would the docs then tell you to call them using `$this->` https://phpunit.readthedocs.io/en/8.4/writing-tests-for-phpunit.html? I've never seen test cases that call assert's as static method calls. E.g. https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Mime/Tests/EmailTest.php#L43

Reply:
I do understand the confusion as they are defined as static functions in `Assert.php`.

Reply:
I'm sorry, I do this more to follow some IDE inspections that I have.
![image](https://user-images.githubusercontent.com/1641075/66474850-6b6de600-ea92-11e9-868a-1755b5a4bae9.png)

But if it's a big deal I can update the PR to call the assertion methods using `$this->` instead of `self::`.

Reply:
Just a small addition, as stated in https://phpunit.readthedocs.io/en/8.4/assertions.html#appendixes-assertions-static-vs-non-static-usage-of-assertion-methods using assertion methods statically or not, is a matter of preference. Therefore since you prefer it like this on your lib I updated my PR to use `$this` instead.